Service/XxllncToZGWZaakTypeService.php

```diff
@@ -0,0 +1,376 @@
+<?php
+
+namespace CommonGateway\XxllncZGWBundle\Service;
+
+use App\Entity\Entity as Schema;
+use App\Entity\Gateway as Source;
+use App\Entity\Mapping;
+use App\Entity\ObjectEntity;
+use App\Service\SynchronizationService;
+use CommonGateway\CoreBundle\Service\CallService;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\Persistence\ObjectRepository;
+use Exception;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class XxllncToZGWZaakTypeService
+{
+    private EntityManagerInterface $entityManager;
+    private CallService $callService;
+    private SynchronizationService $synchronizationService;
+    private SymfonyStyle $io;
+    private array $configuration;
+    private array $data;
+
+    private ?ObjectRepository $objectRepo;
+    private ?ObjectRepository $schemaRepo;
+    private ?ObjectRepository $sourceRepo;
+    private ?ObjectRepository $mappingRepo;
+
+    private ?Source $xxllncAPI;
+    private ?Schema $zaakTypeSchema;
+    private ?Schema $rolTypeSchema;
+    private ?Mapping $caseTypeMapping;
+
+    private ?ObjectEntity $catalogusObject;
+
+    private array $skeletonIn;
+
+    /**
+     * @param EntityManagerInterface $entityManager
+     */
+    public function __construct(
+        EntityManagerInterface $entityManager,
+        SynchronizationService $synchronizationService,
+        CallService $callService
+    ) {
+        $this->entityManager = $entityManager;
+        $this->synchronizationService = $synchronizationService;
+        $this->callService = $callService;
+
+        $this->objectRepo = $this->entityManager->getRepository(ObjectEntity::class);
+        $this->schemaRepo = $this->entityManager->getRepository(Schema::class);
+        $this->sourceRepo = $this->entityManager->getRepository(Source::class);
+        $this->mappingRepo = $this->entityManager->getRepository(Mapping::class);
```

Comment:
I don't like doing this^ (just my opinion), but can't come up with a reason why we shouldn't do this right now.

Reply:
->getRepository('App:Entity');
@bbrands02